.gitignore

```diff
@@ -13,3 +13,4 @@
 .ycm_extra_conf.py
 apex
 packages
+.vscode
```

Comment:
This is likely accidental. It can go in if you want, but add back the newline in that case.